src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,77 @@
+package duke.task;
+
+/**
+ * Represents a task of the user.
+ *
+ * @author wz2k
+ */
+public abstract class Task {
+    /** Task description */
+    private String description;
+
+    /** Marking to show if task is done */
+    private boolean isMarked;
+
+    /**
+     * Creates a task.
+     *
+     * @param description Description of the task.
+     * @param isMarked Task marking.
+     */
+    public Task(String description, boolean isMarked) {
+        this.description = description;
+        this.isMarked = isMarked;
+    }
+
+    /**
+     * Returns the symbol to show if the task is done or not.
+     *
+     * @return Task marked symbol.
+     */
+    private String isMarkedSymbol() {
```

Comment:
As it not returning a boolean, perhaps it could have a 'get' verb in in the method name